source/ccci-stolaf-college/v1/gh-pages.ts

```diff
@@ -1,2 +1,13 @@
-export const GH_PAGES = (filename: string) =>
-	new URL(`https://stodevx.github.io/AAO-React-Native/${filename}`)
+const BASE_URL = 'https://stodevx.github.io'
+
+export const GH_PAGES = (filename: string) => {
+	const url = new URL(BASE_URL)
+	url.pathname = `/AAO-React-Native/${filename}`
+	return url
+}
+
+export const GH_PAGES_FROM_REPO = (reponame: string, filename: string, ) => {
```

Comment:
I'm not sure why the linter didn't flag the trailing comma here. We'll have to investigate.